.github/workflows/generate-types.yml

```
@@ -1,7 +1,7 @@
 name: Generate Types
 
 on:
-  workflow_dispatch:
+  workflow_call:
```

Keep `workflow_dispatch` available here:

```suggestion
  workflow_call:
  workflow_dispatch:
```